Makefile

```
@@ -0,0 +1,7 @@
+.PHONY: post
```

特別理由がなければ、 `.PHONY` は一箇所に集約しても良いかと思います。

```suggestion
.PHONY: post taglist
```

---

なるべくコマンド近くに書いています

---

承知しました。